src/labels_by_user_input.py

```diff
@@ -1,21 +1,42 @@
+import sys
+
 from src.utils import get_labels
 
 
-LABEL_VERIFIED = "Verified"
+LABELS_SET = {
+    "hold": "do-not-merge",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+
+def remove_label(pull, label):
+    print(f"Remove {label} from {pull.title}")
+    pull.remove_from_labels(label)
 
 
 def remove_verified_label(pull):
     labels = get_labels(pull=pull)
-    if LABEL_VERIFIED in labels:
-        print(f"Remove {LABEL_VERIFIED} from {pull.title}")
-        pull.remove_from_labels(LABEL_VERIFIED)
+    if LABELS_SET["verified"] in labels:
+        remove_label(pull=pull, label=LABELS_SET["verified"])
 
 
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    if "/verified" in body and LABEL_VERIFIED not in get_labels(pull=pull):
-        print(f"Adding {LABEL_VERIFIED} to {pull.title}")
-        pull.add_to_labels(LABEL_VERIFIED)
+    labels = get_labels(pull=pull)
+
+    if body[0] == "/":
+        label = body[1:].lower()
+
+        try:
+
+            if label[:2] == "un" and LABELS_SET[label[2:]] in labels:
```

Comment:
I don't like the string cuts [:2], it's not understood
can we think on a better why to do it?

Reply:
I was under the impression regex was just over complicating it.
Should I just use regex, does this look good:
`if "un" in label:
    if any(LABELS.keys()) in label:`
    
Or a variation which combines addition and removal